README.md

```diff
@@ -201,6 +212,14 @@ The following documentation is available:
 <a name="known-issues"></a>
 ## Known Issues
 
+<a name="bugs"></a>
+### Bugs
+
+For a list of current bugs, see the [release notes](https://github.com/minishift/minishift/releases).
+
+<a name="unsupported-features"></a>
+### Unsupported features
```

Comment:
I think we have to say this different. "Unsupported features" to me means, there is something which one can use which is not supported by some sort of support contract. This does not make sense from a community perspective and is not really what this is about.